lib/gds-sso/railtie.rb

```diff
@@ -1,6 +1,10 @@
 module GDS
   module SSO
     class Railtie < Rails::Railtie
+      config.action_dispatch.rescue_responses.merge!(
+        "GDS::SSO::PermissionDeniedError" => 403,
```

Comment:
```suggestion
        "GDS::SSO::PermissionDeniedError" => :forbidden,
```

To match how Rails configures these

Reply:
oops i changed that when i was testing